docker-compose.yaml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: routemaster
+
+  serve:
+    build: .
+    image: routemaster:dev
+    entrypoint:
+      - routemaster
+      - --config-file=config.dev.yaml
+      - serve
+      - --debug
+    volumes:
+      - .:/routemaster/app
+    ports:
+      - "2017:2017"
+    working_dir: /routemaster/app
+    environment:
+      DB_NAME: routemaster
+      DB_USER: routemaster
+      DB_HOST: db
+      DB_PORT: 5432
+    depends_on:
+      - db
+
+  migrate:
```

Comment:
@danpalmer Instead of having repetition and multiple images built for different tasks, you can have a single service for the routemaster app with a sensible default command (serve the app), and one-off commands can be run with something like `docker-compose run --rm routemaster CUSTOM_COMMAND`.

Reply:
Thanks! I've simplified this on master.